soc/egis/et171/start.S

```diff
@@ -22,7 +22,11 @@ SECTION_FUNC(init, entry)
 	.word 0x53494745 /* "EGIS" */
 	.word CONFIG_SOC_EGIS_ET171_FIRMWARE_VERSION /* version */
 	.word CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET /* flash start address */
+#if CONFIG_SOC_EGIS_ET171_VERNDOR_FIRMWARE_SIZE
```

Comment:
Please use `#ifdef CONFIG_SOC_EGIS_ET171_VERNDOR_FIRMWARE_SIZE > 0` to be super clear.